aptos-move/aptos-vm/src/data_cache.rs

```diff
@@ -114,7 +123,7 @@ impl<'a, S: StateView> StorageAdapter<'a, S> {
             }
             let group_data = self.get_resource_group_data(address, &resource_group)?;
             if let Some(group_data) = group_data {
-                let len = if self.accurate_byte_count {
+                let mut len = if self.accurate_byte_count {
```

Comment:
we probably should merge this like
```
if self.accurate_byte_count {
...
} else if self.grouop_byte_count_as_sum {
...
} else {
  0
}
```

Reply:
yeah, makes sense, didn't do because the group_data name was being re-assigned, let me use a different name and do together